### DISCUSSION TITLE
How to install WT-Tracker ? (Windows)

### DISCUSSION THREAD

**Discussion by TheAntinormal** [Novage/wt-tracker#69] [General]
Hello,

Sorry, this is probably a stupid question, but I have issues installing WT-Tracker on my machine..

1. I download the repository as zip.
2. I unpack the zip file, and open a cmd in the root folder.
3. I run npm install.

And while installing, I got a HUGE list like this of missing files.. 
![image](https://user-images.githubusercontent.com/25394460/125292286-9ac61c80-e322-11eb-9052-206e2311a908.png)

Have I done something wrong ? 

Thanks !

**Comment by ckcr4lyf** [comment]
What version of nodejs and typescript do you have? 

I'd advise you to use nodejs LTS (v14.17.1 i think), and update typescript as well ( `npm i -g typescript@latest` ) Then try it again

**Comment by TheAntinormal** [comment]
> What version of nodejs and typescript do you have?
> 
> I'd advise you to use nodejs LTS (v14.17.1 i think), and update typescript as well ( `npm i -g typescript@latest` ) Then try it again

Hello,

Thanks for your answer ! 

Well, I've updated node to v14.17.3, & typescript using the command you gave, then tried again and the same thing happened. 

It seems like there's folders or stuff missing from this repo (like a "dist/" folder) & therefore I can't install it.. :/

**Comment by renatofrota** [comment]
You are missing git. Install git and you will be able to pass this step.
